### PR TITLE
Update index page of Command for GA

### PR DESCRIPTION
**Pull request by thomas11**
Basically
- remove mentions of "preview",
- add code for the examples in other languages, except the EC2/copy one where I don't have tested code at this point,
- minor fixes.

Regarding the EC2/copy example: it spends about ~22 lines on provisioning an EC2 instance with pulumi-aws, with no usage of Command. I think it might be good to remove that part. I have working code in all languages that assumes the remote server exists and takes its IP as a config.

### CODE REVIEW

File: themes/default/content/registry/packages/command/_index.md
Comment (mjeffryes) [inline themes/default/content/registry/packages/command/_index.md:7]: Nit. I would suggest placing this after the first sentence and maybe rephrasing as "The package also includes a resource for copying assets to remote hosts via SSH"

**Review by mjeffryes — APPROVED** [review]

**Review by cnunciato — APPROVED** [review]
Hey @thomas11, I noticed the chooser was kind of broken and the JS examples didn't quite work, so I took the liberty of pushing a commit to save you the hassle of fixing these things up. Hope that's ok!

Approving to unblock, but re this one:

>Regarding the EC2/copy example: it spends about ~22 lines on provisioning an EC2 instance with pulumi-aws, with no usage of Command. I think it might be good to remove that part. 

I agree -- and what's worse it that the relevant parts aren't even visible in the snippet because it's being cropped. Definitely support swapping this out with something else, but will leave that to you.

How about the "Using local.Command with CURL to manage external REST API" example? Do you have code in the other languages for that one?

File: themes/default/content/registry/packages/command/_index.md
Comment (cnunciato) [reply themes/default/content/registry/packages/command/_index.md:172]: Ah, looks like the affected examples will be captured in https://github.com/pulumi/examples/pull/1654. 

Filed this for the content stuff: https://github.com/pulumi/docs/issues/12166

Feel free to add anything else to that list that seems relevant. (Just want to make sure we cover any breaking changes beyond `CopyFile`.)

Comment (thomas11) [reply themes/default/content/registry/packages/command/_index.md:172]: `CopyFile` still works, it just has a deprecation notice. This is also explained in the blog post. So IMO we don't need anything else here.

**Comment by thomas11** [comment]
> Hey @thomas11, I noticed the chooser was kind of broken and the JS examples didn't quite work, so I took the liberty of pushing a commit to save you the hassle of fixing these things up. Hope that's ok!

Thank you, I appreciate it!

> > Regarding the EC2/copy example: it spends about ~22 lines on provisioning an EC2 instance with pulumi-aws, with no usage of Command. I think it might be good to remove that part.
> 
> I agree -- and what's worse it that the relevant parts aren't even visible in the snippet because it's being cropped. Definitely support swapping this out with something else, but will leave that to you.

I swapped it out. The actual copy and command are still the same.

> How about the "Using local.Command with CURL to manage external REST API" example? Do you have code in the other languages for that one?

That's the only one remaining where I don't have code in other languages yet.

**Pull request merged**
